e2e/images/build.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+set -e
+
+DIR=$(dirname "$0")
+cd $DIR
+
+function push {
+    for IMAGE_NAME in $(find * -name Dockerfile -exec dirname {} \; | tr '/' '-')
+    do
+        docker push docker.io/kedacore/$IMAGE_NAME:latest
```

Comment:
I'm not sure about the naming for these images. this change currently assumes that any folder in `e2e/images` that contains a `Dockerfile` is an "image". So for example

```
├── artemis
│   ├── consumer
│   │   └── Dockerfile
│   └── producer
│       └── Dockerfile
├── azure-blob
│   └── Dockerfile
├── kafka
│   └── Dockerfile
```

results in 

```
docker.io/kedacore/artemis-consumer
docker.io/kedacore/artemis-producer
docker.io/kedacore/azure-blob
docker.io/kedacore/kafka
```


I don't like polluting the top level repo there, but not sure what's the best alternative.

Reply:
LGTM

Reply:
But latest is very hard to troubleshoot, can we use commit or version tag instead?

Reply:
LGTM, maybe add `tests-` or `e2e-` prefix to the image names?

Reply:
@tomkerkhove I don't expect big changes in those images and they are used in one place, so `latest` is not a big deal. But adding a version on anything could do any harm, right? 😄

Reply:
I made the abbreviated commitid be the tag for the images.